### ISSUE TITLE
Multibrand example still uses templates rather than formats

### ISSUE THREAD

**Issue by custa1200** [amzn/style-dictionary#226]
In the build.js file the android and iOS groups still use the old template formats. It also uses a mix of quotation marks as can be seen in the build path.

```
     "android": {
        "transformGroup": "android",
        "buildPath": `build/android/${brand}/`,
        "files": [{
          "destination": "tokens.colors.xml",
          "template": "android/colors"
        },{
          "destination": "tokens.dimens.xml",
          "template": "android/dimens"
        },{
          "destination": "tokens.font_dimens.xml",
          "template": "android/fontDimens"
        }]
      },
      "ios": {
        "transformGroup": "ios",
        "buildPath": `build/ios/${brand}/`,
        "files": [{
          "destination": "tokens.h",
          "template": "ios/macros"
        }]
      }

```

**Comment by custa1200** [comment]
Sorry I saw why you have used that style of quotation marks there, the script breaks if using traditional quotes. Ignore the second bit of my issue :)

**Comment by didoo** [comment]
@custa1200 I'll have a look at the example (I've done it, I don't remember if there were good reasons to still use templates in this case). Regarding the quotes, they are [template literals](https://github.com/amzn/style-dictionary/issues/226), they are intended :)

**Comment by custa1200** [comment]
> Regarding the quotes, they are template literals, they are intended :)

Well thats my thing learnt for the day :)

**Comment by didoo** [comment]
@custa1200 I have checked, and the example with the multi-brand example is correct, the "template" is still a valid declaration in the config (these are the standard templates). what is deprecated is the "registerTemplate" method, which actually was still used in another example, so I have updated that one in #232.

Thank you for raising the issue.

**Comment by dbanksdesign** [comment]
To add a bit more context, when we began the deprecation path of templates, we copied all built-in templates to formats of the same name. Until we full remove templates/registerTemplate from the codebase you should be able to call them either as formats or templates. If you call it as a template it will show a warning message and how to migrate though!

**Comment by chazzmoney** [comment]
Fixed in v2.7.0
